en/docs/observe/mi-observe/working-with-monitoring-dashboard.md

```diff
@@ -252,6 +252,15 @@ Now you can view details of artifacts, update artifacts, and perform various oth
 
 <a href="{{base_path}}/assets/img/integrate/monitoring-dashboard/dashboard-artifact-list.png"><img src="{{base_path}}/assets/img/integrate/monitoring-dashboard/dashboard-artifact-list.png" width="300"></a>
 
+## Monitoring health of MI Dashboard
+MI Monitoring Dashboard provide a health check endpoint that can be used to monitor the status of the Dashboard. The following API format can be used to invoke the health API:
+```
+https://localhost:9743/dashboard/api/healthz
```

Comment:
@madushadhanushka , is the "z" at the end of the URL correct?

Reply:
yes. This link is same as MI health check endpoint URL